rest/accounts/list-get-example-1/list-get-example-1.5.x.cs

```diff
@@ -0,0 +1,21 @@
+// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
+using System;
+using Twilio;
+using Twilio.Rest.Api.V2010;
+
+class Example
+{
+    static void Main(string[] args)
+    {
+        // Find your Account Sid and Auth Token at twilio.com/console
+        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string authToken = "your_auth_token";
+        TwilioClient.Init(accountSid, authToken);
+
+        var accountsResourceSet = AccountResource.Read();
```

Comment:
Please use `accounts` instead of `accountsResourceSet`. Generally, we don't need to append the type of the variable, that's some kind of Hungarian notation.